regress/pesterTests/Uninstall.Tests.ps1

```diff
@@ -0,0 +1,196 @@
+﻿If ($PSVersiontable.PSVersion.Major -le 2) {$PSScriptRoot = Split-Path -Parent $MyInvocation.MyCommand.Path}
+Import-Module $PSScriptRoot\CommonUtils.psm1 -Force
+$suite = "Uninstall"
+$tC = 1
+$tI = 0
+Describe "Uninstall Tests" -Tags "Uninstall" {
+    BeforeAll {
+        if($OpenSSHTestInfo -eq $null)
+        {
+            Throw "`$OpenSSHTestInfo is null. Please run Set-OpenSSHTestEnvironment to set test environments."
+        }      
+        
+        $windowsInBox = $OpenSSHTestInfo["WindowsInBox"]
+        $binPath = $OpenSSHTestInfo["OpenSSHBinPath"]
+        $dataPath = Join-path $env:ProgramData ssh        
+
+        Stop-Service sshd -ErrorAction SilentlyContinue
+        Stop-Service ssh-agent -ErrorAction SilentlyContinue
+        if(Get-Service sshd -ErrorAction SilentlyContinue)
+        {        
+            if($windowsInBox) {
+                Remove-WindowsCapability -online -name OpenSSH.Server~~~~0.0.1.0
```

Comment:
Please remove 0.0.1.0 and use regex as our version (0.0.1.0) keeps on changing and we don't want to modify our scripts.

Reply:
We previously confirmed with fod team. This version won't changed. If changes, it will be a different fod